src/artofillusion/tapDesigner/TaPDObjectMaterialDialog.java

```diff
@@ -146,12 +148,13 @@ private void doCancel()
   
   private void doNewMaterial()
   {
-    MaterialsDialog.showNewMaterialWindow(this, sc);
+      JOptionPane.showMessageDialog(this.component, "MaterialsDialog class and showNewMaterialWindow method deleted.");
```

Comment:
As a short-term fix, these should probably just create a `new TexturesAndMaterialsDialog(this, sc);`

We can clean up the extra UI elements later.

Reply:
AOI's TexturesAndMaterialsDialog required EditingWindow and Scene as constructor parameters so it cannot be directly used here.
... Same for all other comments

Reply:
Ahh... that's right. Something else to think about.

I'm not sure why TaPD does all of this extra management of Textures and Materials, I guess we can let that be broken for a bit.

Reply:
What's not clear is why TaPD doesn't implement `EditingWindow`... it seems to have taken some trouble to break away from AOI core classes in some places, not sure why.